doc/nrf/app_dev/device_guides/nrf54l/features.rst

```diff
@@ -14,10 +14,29 @@ For additional information, see the following documentation:
 * Zephyr page on the :ref:`zephyr:nrf54l15dk_nrf54l15`
 * :ref:`installation` and :ref:`configuration_and_build` documentation to install the |NCS| and learn more about its development environment.
 
+RISC-V core
+***********
+
+VPR core, named *fast lightweight peripheral processor* (FLPR), is designed to support the RISC-V instruction set, featuring key enhancements that enable efficient handling of basic and complex operations, as well as streamlined instruction processing.
```

Comment:
```suggestion
The nRF54L15 DK has VPR core named *fast lightweight peripheral processor* (FLPR).
It is designed to support the RISC-V instruction set, and features key enhancements that enable efficient handling of basic and complex operations, as well as streamlined instruction processing.
```
Both VPR and FLPR could also have the term formatting added to them, but that's not something that has to be done.